run_afs_to_pcdshub.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python -m afs_ioc_migration --org pcdshub --dry-run /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/*/*
```

Comment:
Since the dry-run creates the repos, perhaps we should have an option to simply commit the repos that are in a specific space?  So we don't have to re-run all the repo-migration steps if we're satisfied (this might also let the user make a few small tweaks before committing?)

I wonder if it's worth adding some kind of "pickup where you left off" functionality.  If github decides to rate-limit you mid way, there's currently no way of saying "skip x repos" right?  I see that we skip repos if they exist and have commits in github, but we'd still end up re-making the repo in a temp drive right?

Reply:
This is a fair point, it's more or less a full one-and-done shot right now. The order is important (lock -> clone -> modify -> push), so I don't think the dry-run output should be committed (it could be out of date) but the timing is less important...

Reply:
We actually won't be rate-limited unless we run this a third time in an hour btw, there's enough per-hour quota to run it twice (600 repos * 3 API calls = 1800 calls < 5000 call/hr limit)

Reply:
I think declaring that the whole process be done at once is a reasonable stance to take.  Definitely not worth losing any sleep over

Reply:
Also to be clear my concern here wasn't exclusively about rate limiting, there are other more unlikely things that can happen, like the internet going out 😆